filemonitor/FileMonitor.hpp

```diff
@@ -8,22 +8,25 @@
 #include <chrono>
 #include <thread>
 
+// Macro definition for time period 500
+#define TIMEPERIOD 500
+
 namespace fs = std::filesystem;
 
-class FileMonitor : public IFileMonitor {
+class FileMonitor final : public IFileMonitor {
     public:
-        FileMonitor(const std::string& filePath) : filePath(filePath) {}
+        FileMonitor(const std::string& m_filePath) : m_filePath(m_filePath) {}
```

Comment:
please don't use similar names for function arguments and variables

Reply:
ok, corrected